telefunc/plugin/next/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { NextConfig } from 'next'
+import { resolve } from 'path'
+
+export default plugin
+
+function plugin(nextConfig: NextConfig = {}): NextConfig {
+  return Object.assign({}, nextConfig, {
+    webpack: (config, options) => {
+      config.module.rules.push({
+        test: /\.telefunc\./,
+        use: [{ loader: resolve(__dirname, './loader.js') }],
+      })
+
+      config.module.rules.push({
+        test: /api(\/|\\)_telefunc/,
```

Comment:
This test is flaky, imagine `/pages/api/_telefunc.page.mdx` (which actually may happen for `https://telefunc.org`.) Admittedly it's unlikely to happen, but these little errors do cumulate over time.

Reply:
Let's keep this for later!